libmisc/salt.c

```diff
@@ -283,7 +283,7 @@ static /*@observer@*/const char *YESCRYPT_salt_cost (/*@null@*/long *prefered_co
         long cost;
```

Comment:
Also inconsistent indentation in the above two lines. (I didn't realize you wouldn't see all of them at once.)